internal/worker.go

```diff
@@ -95,39 +82,6 @@ type (
 		// default: false not to heartbeat.
 		AutoHeartBeat bool
 
-		// Optional: Sets an identify that can be used to track this host for debugging.
-		// default: default identity that include hostname, groupName and process ID.
-		Identity string
-
-		// Optional: Metrics to be reported. Metrics emitted by the temporal client are not prometheus compatible by
-		// default. To ensure metrics are compatible with prometheus make sure to create tally scope with sanitizer
-		// options set.
-		// var (
-		// _safeCharacters = []rune{'_'}
-		// _sanitizeOptions = tally.SanitizeOptions{
-		// 	NameCharacters: tally.ValidCharacters{
-		// 		Ranges:     tally.AlphanumericRange,
-		// 		Characters: _safeCharacters,
-		// 	},
-		// 		KeyCharacters: tally.ValidCharacters{
-		// 			Ranges:     tally.AlphanumericRange,
-		// 			Characters: _safeCharacters,
-		// 		},
-		// 		ValueCharacters: tally.ValidCharacters{
-		// 			Ranges:     tally.AlphanumericRange,
-		// 			Characters: _safeCharacters,
-		// 		},
-		// 		ReplacementCharacter: tally.DefaultReplacementCharacter,
-		// 	}
-		// )
-		// opts := tally.ScopeOptions{
-		// 	Reporter:        reporter,
-		// 	SanitizeOptions: &_sanitizeOptions,
-		// }
-		// scope, _ := tally.NewRootScope(opts, time.Second)
-		// default: no metrics.
-		MetricsScope tally.Scope
-
 		// Optional: Logger framework can use to log.
 		// default: default logger provided.
 		Logger *zap.Logger
```

Comment:
Why logger is not moving?

Reply:
I removed only those which exist in `ClientOptions` already. `Logger` is not there. Apparently, client doesn't need it.